notebooks/book1/03/sensor_fusion_2d.ipynb

```diff
@@ -0,0 +1,503 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #68.                    "label": "$E[x|y_1; y_2]$",
</pre><p>This should use E[x|y_1, y_2] as per the book.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Yes will fix that